v3/src/data-interactive/resource-parser-utils.ts

```diff
@@ -0,0 +1,41 @@
+import { IDataSet } from "../models/data/data-set"
+import { DIParsedQuery, DIQueryFunction } from "./data-interactive-types"
+
+export function parseSearchQuery(query: string, dataContext?: IDataSet): DIParsedQuery {
+  if (query === "*") {
+    return { valid: true, func: () => true }
+  }
+
+  const matches = query.match(/([^=!<>]+)(==|!=|<=|<|>=|>)([^=!<>]+)/)
```

Comment:
I assume this parsing code is taken from v2?

Reply:
Yes. Also on line 14, though there was a bug in that code in v2 that chatGPT helped me fix 👍  Should I add a comment mentioning where this code came from?

Reply:
Seems like it could be useful.